### PR TITLE
Add batch subsystem capabilities

### PR DESCRIPTION
**Pull request by jamezp**

### CODE REVIEW

File: org/wildfly/batch/default/job/repository/capability.adoc [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+DYNAMIC: false
+
+PRIVATE: true
+
+SUPPORTS RUNTIME ONLY: false
+
+SERVICE PROVIDED:
+
+  CLASS: org.jberet.spi.JobExecutor
+  MODULE: org.jberet.repository.JobRepository
```
Comment (bstansberry) [inline org/wildfly/batch/default/job/repository/capability.adoc]: org.jberet.jberet-core ?

File: org/wildfly/batch/job/repository/capability.adoc [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+DYNAMIC: true
+
+PRIVATE: true
+
+SUPPORTS RUNTIME ONLY: false
+
+SERVICE PROVIDED:
+
+  CLASS: org.jberet.spi.JobExecutor
+  MODULE: org.jberet.repository.JobRepository
```
Comment (bstansberry) [inline org/wildfly/batch/job/repository/capability.adoc]: org.jberet.jberet-core ?

File: org/wildfly/batch/thread/pool/capability.adoc [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+DYNAMIC: true
+
+PRIVATE: true
+
+SUPPORTS RUNTIME ONLY: false
+
+SERVICE PROVIDED:
+
+  CLASS: org.jberet.spi.JobExecutor
+  MODULE: org.jberet.jberet-core
```
Comment (bstansberry) [inline org/wildfly/batch/thread/pool/capability.adoc]: org.jberet.jberet-core ?

**Comment by jamezp** [comment]
Oops, should be fixed.

File: org/wildfly/batch/job/repository/capability.adoc [hunk 4]
```diff
@@ -0,0 +1,24 @@
+NAME: org.wildfy.batch.job.repository
```
Comment (claudio4j) [inline org/wildfly/batch/job/repository/capability.adoc]: A typo in wildfy

**Pull request merged**